centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/Columns/HoverChip.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from 'react';
+
+import { not } from 'ramda';
+
+import { Tooltip } from '@mui/material';
+
+import { useMemoComponent } from '@centreon/ui';
+
+import { useHoverChiptStyles } from './Columns.styles';
+
+interface Props {
+  Chip: () => JSX.Element;
+  children: (props?) => JSX.Element;
+  isHovered?: boolean;
+  label: string;
+  onClick?: () => void;
+}
+
+const HoverChip = ({
+  children,
+  Chip,
+  label,
+  onClick,
+  isHovered = false
+}: Props): JSX.Element => {
+  const { classes } = useHoverChiptStyles();
+
+  const [isChipHovered, setIsChipHovered] = useState<boolean>(false);
+
+  const openTooltip = (): void => setIsChipHovered(true);
+
+  const closeTooltip = (): void => setIsChipHovered(false);
+
+  useEffect(() => {
+    if (not(isHovered)) {
+      return;
+    }
+    setIsChipHovered(false);
+  }, [isHovered]);
+
+  return useMemoComponent({
+    Component: (
+      <Tooltip
+        PopperProps={{
+          onClick: (e): void => {
+            e.preventDefault();
+            e.stopPropagation();
+          }
+        }}
+        aria-label={label}
+        classes={{ tooltip: classes.tooltip }}
+        enterDelay={200}
+        enterNextDelay={200}
+        leaveDelay={0}
+        open={isChipHovered}
```

Comment:
Why are you using a controlled state for this?
Make it uncontrolled, isn't enough?

Reply:
I'm not sure what you mean, but this state is utilized to display detailed information about a state in a table when the cursor hovers over the State column